profile.md

```diff
@@ -37,7 +37,16 @@ _Authored by the WLCG AuthZ Working Group_
 
 # Introduction
 
-This document describes how WLCG users may use the available geographically distributed resources without X.509 credentials.  In this model, clients are issued with bearer tokens; these tokens are subsequently used to interact with resources.  The tokens may contain authorization groups and/or capabilities, according to the preference of the VO, applications and relying parties. 
+This document describes how WLCG users may use the available geographically distributed resources without X.509 credentials.  In this model, clients are issued with bearer tokens; these tokens are subsequently used to interact with resources.  The tokens may contain authorization groups and/or capabilities, according to the preference of the virtual organisation, applications and relying parties.
+
+A virtual organisation (VO) is a group of users that are
+self-managing, often with internal structure (such as groups and
+roles).  The members of a VO work towards a common goal; for example,
+maximising the scientific output from a specific scientific
+instrument.  A VO has well understood membership criteria, which the
```

Comment:
```suggestion
instrument  or providing shared services to multiple related projects.
A VO has well understood membership criteria, which the
```

Reply:
Thanks for the suggestion.  I've updated the text.

I was wondering if you have a good example of a service-providing VO, that I include in the list of example VOs?

Reply:
`fermilab` and `osg` are the two that I know about.

Reply:
Thanks